templates/provider/kcm/templates/rbac/controller/roles.yaml

```diff
@@ -95,6 +95,8 @@ rules:
   resources:
   - managements
   - regions
+  - datasources
```

Comment:
We probably need to add an extra user-facing role for `datasources` to the kcm provider template.